<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-L8VPHJS" failures="1" tests="5" name="testScripts.ExtentReportSampleTest" time="24.527" errors="0" timestamp="2024-11-11T10:33:18 IST" skipped="3">
  <testcase classname="testScripts.ExtentReportSampleTest" name="cucumberSearchTest" time="5.947">
    <skipped/>
  </testcase> <!-- cucumberSearchTest -->
  <system-out/>
  <testcase classname="testScripts.ExtentReportSampleTest" name="cucumberSearchTest" time="4.658">
    <skipped/>
  </testcase> <!-- cucumberSearchTest -->
  <system-out/>
  <testcase classname="testScripts.ExtentReportSampleTest" name="cucumberSearchTest" time="4.620">
    <failure message="expected [Cucumber Tutorial - Google Search] but found [Cucumber Tutorial - Google Search Page]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Cucumber Tutorial - Google Search] but found [Cucumber Tutorial - Google Search Page]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testScripts.ExtentReportSampleTest.cucumberSearchTest(ExtentReportSampleTest.java:63)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:214)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- cucumberSearchTest -->
  <system-out/>
  <testcase classname="testScripts.ExtentReportSampleTest" name="cucumberSearchTest" time="4.702">
    <skipped/>
  </testcase> <!-- cucumberSearchTest -->
  <system-out/>
  <testcase classname="testScripts.ExtentReportSampleTest" name="appiumSearchTest" time="4.600"/>
  <system-out/>
</testsuite> <!-- testScripts.ExtentReportSampleTest -->
